Makefile

```diff
@@ -55,7 +55,7 @@ all: build test
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildSHA "$(shell git rev-parse HEAD)"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTag "$(shell git describe --dirty)"
 build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildTime "$(shell date -u '+%Y/%m/%d %H:%M:%S')"
-build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildDeps "$(shell make depvers | sort)"
+build: LDFLAGS += -X github.com/cockroachdb/cockroach/util.buildDeps "$(shell GOPATH=${GOPATH} build/depvers.sh)"
```

Comment:
No, it is still buildDeps. The buildVers variable holds the Go version (e.g. "go1.4.2").